backstory/src/main/webapp/style.css

```diff
@@ -0,0 +1,106 @@
+/* General */
+
+html,
+body {
+  height: 100%;
```

Comment:
Was `make pretty` okay with h ordered above f? If so, maybe the linter is not working.

Reply:
I ran the old ```make pretty``` on this. The new make pretty errors out on this (even after the merge) with: ```** [makefile:15: pretty] Error 127``` because it can't find a file (```xargs: node_modules/stylelint/bin/stylelint.js: No such file or directory```). I ran it right after merging with the master, so I can't be sure.

Reply:
While the linter may not work from the command line, the workflow steps you added pre-upload seem to be working, so that's good.

Reply:
Update: this does work from the command-line, I just had to install the stylelint.

Reply:
Done